core/src/services/azdls/core.rs

```diff
@@ -273,23 +229,21 @@ impl AzdlsCore {
             .trim_end_matches('/')
             .to_string();
 
-        let mut url = format!(
+        let mut url = QueryPairsWriter::new(&format!(
```

Comment:
QueryPairsWriter requires the base must not have query. Please put `resource` and `recursive` seperately.

Reply:
fixed